adam/curriculum/phase1_curriculum.py

```diff
@@ -354,6 +355,9 @@ def make_fall_templates() -> Iterable[Phase1SituationTemplate]:
             syntax_hints=[USE_ADVERBIAL_PATH_MODIFIER]
             if use_adverbial_path_modifier
             else [],
+            before_action_relations=[
+                rel.negated_copy() for rel in on(arbitary_object, ground)
```

Comment:
You can do just `negate(on(arbitrary_object, ground))`.

Reply:
Fixed.